### DISCUSSION TITLE
Upload osu!lazer on Play Store and App Store

### DISCUSSION THREAD

**Discussion by itsskyballs** [ppy/osu#21483] [Ideas] [answered]
Some People want to Download lazer on their Mobile Phone
but they didn't know Where to Download lazer

**Comment by bdach** [comment, marked as answer]
https://twitter.com/ppy/status/1587065060186558464

**Reply by itsskyballs** [reply]
you can upload lazer on google firebase by copying the repository

**Reply by bdach** [reply]
With all due respect, I don't think you know what you're talking about.

And as the link explains, the mechanics of putting the game on the appstore aren't the problem. The associated support requirements that would come with that are the problem. So suggesting "firebase" as a solution to anything is missing the forest for the trees.
